cli/flox-rust-sdk/src/models/environment/core_environment.rs

```diff
@@ -1068,6 +1128,12 @@ pub enum CoreEnvironmentError {
     /// Tried to uninstall a package that wasn't installed
     #[error("couldn't uninstall '{0}', wasn't previously installed")]
     PackageNotFound(String),
+    /// Tried to uninstall a package that was only provided by an include.
+    #[error(
+        "Cannot remove included package '{0}'
+Remove the package from environment '{1}' and then run 'flox include upgrade'"
```

Comment:
Note for reviewers: I added the `flox include upgrade` part which didn't exist when we originally wrote the ticket because it should short-circuit a surprise when the user modifies an include and it doesn't get reflected in the composing environment.

Reply:
Although they might all need moving and threading through if we move the methods to `Manifest` 😑

Reply:
I'm not entirely following this, but we can check back in on that if/when we move the impls..